arcgis-ios-sdk-samples/Layers/WMS layer (URL)/README.md

```diff
@@ -1,9 +1,33 @@
 # WMS layer (URL)
 
-This sample demonstrates how to open a WMS layer from a URL and display it in a map.
+Display a WMS layer using a WMS service URL.
 
-![](image1.png)
+![WMS layer (URL) sample](wms-layer-url.png)
+
+## Use case
+
+WMS is an OGC standard for displaying maps from images that are dynamically-generated on a web server. WMS is particularly useful for data that changes frequently, contains cartographically complex detail, or requires an open source data standard.
+
+## How to use the sample
+
+The map will load automatically when the sample starts. Pan and zoom to explore.
 
 ## How it works
 
-`AGSWMSLayer` provides an initializer called `init(url:layerNames:)` that takes a WMS service URL and an array of layer names available in the WMS service. The URL must be the URL to the `GetCapabilities` endpoint of the WMS Service. The sample loads the WMS Layer using `load(completion:)` method. And on successful completion, the layer is added to the collection of map's operational layers.
+1. Create an `AGSWMSLayer` specifying the URL of the service and the names of layers you want to display.
+    **Note**: The name comes from the `Name` property, not the `Title` property. On many services, the title is human-readable while the name is a numeric identifier.
+2. After the `AGSWMSLayer` loads, add it to the map's `operationalLayers` array.
+
+## About the data
+
+This sample uses a [U.S. National Weather Service radar map](https://nowcoast.noaa.gov/arcgis/services/nowcoast/radar_meteo_imagery_nexrad_time/MapServer/WMSServer?request=GetCapabilities&service=WMS). Because WMS services generate map images on-the-fly, this layer is always up-to-date with the latest [NOAA NEXRAD radar](https://www.ncdc.noaa.gov/data-access/radar-data/nexrad) observations.
+
+## Relevant API
+
+* AGSMap
+* AGSMapView
+* AGSWMSLayer
+
+## Tags
+
+OGC, web map service, WmsLayer
```

Comment:
WmsLayer seems strange to me, but I'm not sure what is its correct spelling.

Reply:
@yo1995 It's just a tag for styling WMS layers. See [Thad's keyword list](https://devtopia.esri.com/runtime/common-samples/wiki/Keywords-for-sample-doc-topics).